plutus-tx/src/Language/PlutusTx/Prelude/Stage0.hs

```diff
@@ -64,6 +65,86 @@ or = [|| \(l :: Bool) (r :: Bool) -> if l then True else r ||]
 not :: Q (TExp (Bool -> Bool))
 not = [|| \(a :: Bool) -> if a then False else True  ||]
 
+-- | Greater than 
+--
+--   >>> $$([|| $$(gt) 2 1 ||])
+--   True
+--
+gt :: Q (TExp (Int -> Int -> Bool))
+gt = [|| (>) ||]
```

Comment:
You should be able to use the non-symbolic ones in `Builtins`. That way I can delete the hacky typeclass handling entirely ;)

Reply:
I'll look into that!